src/plugins/data_source_management/public/components/popover_button/popover_button.test.tsx

```diff
@@ -22,12 +22,19 @@ describe('Test on PopoverButton', () => {
     const component = mount(
       <DataSourceMenuPopoverButton className="random" label={label} onClick={jest.fn()} />
     );
-    expect(component.find('EuiButtonEmpty').text()).toBe(label);
+    expect(component.find('EuiButton').text()).toBe(label);
   });
 
-  it('calls onClick when button is clicked', () => {
+  // ToDo: Find out if this is actually the correct behavior
```

Comment:
still need to validate?

Reply:
Will find the owner to follow up on this.

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/7674